synapse/handlers/appservice.py

```diff
@@ -233,6 +233,13 @@ def notify_interested_services_ephemeral(
         # Additional context: https://github.com/matrix-org/synapse/pull/11137
         assert isinstance(new_token, int)
 
+        # Ignore to-device messages if the feature flag is not enabled
+        if (
+            stream_key == "to_device_key"
```

Comment:
didn't we (you) invent constants for this?

Reply:
I think I asked about where to put them a few weeks ago. It's been on my mind to convert these to constants once all this in-flight stuff had settled.

Reply:
As per https://github.com/matrix-org/synapse/pull/11215/files#r752237473, I've added a todo comment for this task.